TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,106 @@
+# Self-Review Questionnaire: Security and Privacy
+
+Questions from w3.org/TR/security-privacy-questionnaire
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+If unpartitioned third-party cookies are still in use, this feature does not expose any additional information that is not already available if sites were already using an unpartitioned cross-site cookie.
+If unpartitioned third-party cookies are blocked, this provides sites in third-party contexts the ability to use HTTP cookies that are scoped to a single top-level site.
+
+### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes. This proposal gives the ability for third-party contexts the ability to use a familiar HTTP state primitive for use cases restricted to a users' activity in only the current top-level site.
+
+### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+This proposal requires that cookies which use the Partitioned attribute also have the __Host- prefix.
+This prefix requires cookies be hostname scoped and can only be used in [secure contexts](https://www.w3.org/TR/secure-contexts/).
+The latter requirement protects these cookies from sending PII in plaintext between clients and servers.
+
+### 2.4. How does this specification deal with sensitive information?
+
+See 2.3.
+
+### 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+This proposal extends an existing state mechanism that persists across browsing sessions.
+It imposes different types of size limitations on these types of cookies in order to prevent them from being abused as a cross-site tracking vector.
+For more information for this type of attack, see this section of the explainer.
```

Comment:
Did you lose a link to an explainer section?

Reply:
Whoops, fixed.